components/utils/profiler.js

```diff
@@ -2,7 +2,7 @@ import { Profiler as NativeProfiler } from 'react'
 
 export const Profiler = ({ id, children }) => {
   const _id = `${id || children.type.type.displayName}.${children.key}`
-  // console.log(children)
```

Comment:
This and the above console.log can probably be removed as it looks like they are here just for debugging. But I will let @sarathms comment on if he considers them useful to keep.